app/views/layouts/application.html.erb

```diff
@@ -5,11 +5,12 @@
     <%= render "layouts/rails_default" %>
     <%= render "layouts/shim" %>
   </head>
+
   <body>
     <%= render "layouts/header" %>
-    <div class = "container">
+    <div class="container">
       <% flash.each do |message_type, message| %>
-        content_tag :div, <%= message %>, class: "alert alert-<%= message_type %>"
+        <%= content_tag :div, "#{message}", class: "alert alert-#{message_type}" %>
       <% end %>
     	<%= yield %>
```

Comment:
File này của em đang có 2 vấn đề:
* Dòng 4: remove `()`
* Dòng 15, 16: việc lùi đầu dòng đang pha trộn giữa `space` và `tab`. Cụ thể, em mở file đó lên bằng sublime text, bấm `Ctrl + A` để bôi đen tất cả thì sẽ nhìn được hình ảnh trực quan như này:

![image](https://cloud.githubusercontent.com/assets/22311747/24822978/fd0666ae-1c23-11e7-8ae4-53788606dfc8.png)

Nhìn bằng mắt thường thì dễ nhầm tưởng là dòng 15, 16 đang thẳng cột và sử dụng cùng loại indent với dòng 14, tuy nhiên khi bôi đen thì thấy là chúng khác nhau.

Để khắc phục, em click vào mục `Spaces: 2` của thanh status (vẫn đang ở file này nhé), chọn "Convert Indentation to Spaces", save file rồi đẩy lại nhánh `basic-login` lên github nhé.